test/exceptions/Makefile

```diff
@@ -1,6 +1,11 @@
 include ../common.mak
 
 TESTS:=stderr_msg unittest_assert
+ifeq ($(OS)-$(BUILD),linux-debug)
```

Comment:
Ah I see, you already combine the location info with backtrace_symbols, so it should be fine.